plugins/sqlfluff-templater-dbt/test/templater_test.py

```diff
@@ -338,26 +343,17 @@ def test__templater_dbt_handle_database_connection_failure(
             fname=target_fpath,
             config=FluffConfig(configs=DBT_FLUFF_CONFIG),
         )
-    except Exception as e:
-        if DBT_VERSION_TUPLE >= (1, 0):
-            # In dbt 1.0.0, connection failures raise an exception
-            assert str(e).startswith(
```

Comment:
What I thought was just a change to dbt 1.0.0 turned out to be a demonstration of the error that this PR solves